setup.ps1

```
@@ -0,0 +1,887 @@
+        if (-not $clientExists) {
+            $body = @{
+                enable = $true
+                name = "Decypharr"
+                implementation = "QBittorrent"
+                configContract = "QBittorrentSettings"
+                protocol = "torrent"
+                priority = 1
+                fields = @(
+                    @{ name = "host"; value = "decypharr" },
+                    @{ name = "port"; value = 8282 },
+                    @{ name = "username"; value = "http://$Type:$Port" }
```

PSScriptAnalyzer (1.25.0)

[error] Variable reference is not valid. ':' was not followed by a valid variable name character. Consider using ${} to delimit the name.

(InvalidVariableReferenceWithDrive)